Python 3.11.1 (tags/v3.11.1:a7a450f, Dec  6 2022, 19:58:39) [MSC v.1934 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.

=========== RESTART: C:\Users\DELL\Desktop\python programs\create1.py ==========
hi
hi: 20

= RESTART: C:\Users\DELL\Desktop\python programs\create1.py
hi
hi: 20

= RESTART: C:\Users\DELL\Desktop\python programs\create1.py
hi
hi: 20
hi: 30
hi: 40
hi: 444444444444444

= RESTART: C:\Users\DELL\Desktop\python programs\create1.py
hi
hi: {'a': '20', 'b': '30', 'c': '40', 'd': '444444444444444'}
Traceback (most recent call last):
  File "C:\Users\DELL\Desktop\python programs\create1.py", line 7, in <module>
    my_function(a="20",b="30",c="40",d="444444444444444")
  File "C:\Users\DELL\Desktop\python programs\create1.py", line 4, in my_function
    print("hi:",b)
NameError: name 'b' is not defined
>>> 
= RESTART: C:\Users\DELL\Desktop\python programs\create1.py
hi
20
30
40
444444444444444
>>> 
= RESTART: C:\Users\DELL\Desktop\python programs\create1.py
hi
20 30 40 444444444444444
